lighthouse-core/gather/driver.js

```diff
@@ -1401,6 +1398,15 @@ class Driver {
     return this.sendCommand('Runtime.enable');
   }
 
+  /**
```

Comment:
not sure if this is the best place for this since it's more just turning it on than using it, but it would be nice if there were a comment somewhere to answer "why is this being turned on?" (or maybe more accurately, sometime in the future, "was this important or can I remove this?")